examples/js/loaders/GLTFLoader.js

```diff
@@ -1370,11 +1550,13 @@ THREE.GLTFLoader = ( function () {
 								? THREE.QuaternionKeyframeTrack
 								: THREE.VectorKeyframeTrack;
 
+							var targetName = node.name ? node.name : node.uuid;
```

Comment:
As probably you've noticed, here is not for #10366.

The aim is to give the information to `AnimationMixer` to find the target node from a scene even if node.name is empty (node.name is optional in glTF).

To guarantee to pass the unique specification , always passing uuid could be safer because glTF allows two or more nodes have the same name.

Alternative is to return the mappings of animation root node(=target node) and animation from `GLTFLoader`, like this

    {
        targetNode1: animation1,
        targetNode2: animation2
    }